source/_components/image_processing.facebox_face_detect.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: "Facebox"
+description: "Detect and recognise faces with Facebox."
+date: 2018-05-03 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: machine-box-alpha.png
+ha_category: Image Processing
+featured: false
+ha_release: 0.70
+---
+
+The `facebox` image processing component allows you to detect and recognise faces in a camera image using [Facebox](https://machinebox.io/docs/facebox). The state of the entity is the number of faces detected, and recognised faces are listed in the `matched_faces` attribute. Facebox runs in a Docker container, and it is recommended that you run this container on a machine with a minimum of 2 GB RAM. To minimise the time for Facebox to detect faces in an image using this component, on your machine with Docker run Facebox with:
```

Comment:
It's a new platform (facebox) that has been created, not a new component. Facebox is now a platform under the image_processing component.